function/requirements.txt

```diff
@@ -1,3 +1,4 @@
 fastavro==0.22.4
```

Comment:
does it still work if you remove `fastavro` here?

Reply:
good catch!